application/src/main/scala/com/azavea/franklin/crawler/StacImport.scala

```diff
@@ -2,11 +2,12 @@ package com.azavea.franklin.crawler
 
 import java.nio.file.Paths
 
+import cats.effect.Blocker
 import cats.{Applicative, MonadError}
 import com.azavea.franklin.database.{StacCollectionDao, StacItemDao}
 import doobie.ConnectionIO
 import doobie.implicits._
-import geotrellis.server.stac._
+import com.azavea.stac4s._
```

Comment:
I wonder if we should use this `scalafix` rule: https://github.com/NeQuissimus/sort-imports

Sorting imports manually is a drag, and it looks like with the `blocks` configuration we would be able to do first party, third party, scala lang, and java lang blocks

Reply:
oh I like this, I'll add it - good find!

Reply:
Couldn't get this to work, opened https://github.com/azavea/azavea.g8/issues/4 to figure out how to get it to work